common/config/babel/.babelrc.js

```diff
@@ -77,8 +79,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
         'click-to-call',
         // Mention feature
         'mention',
-        // Feature for call transfer
-        'call-transfer'
```

Comment:
perhaps this got overwritten in a merge? keep this as well as the new change

Reply:
Fixed